### PR TITLE
Service manager/issue 111 tests for activity manager

### PR DESCRIPTION
**Pull request by martikyan**
Solves #111 and moves tests from root

### CODE REVIEW

**Comment by martikyan** [comment]
I have skipped checking for null arguments because no checks were made

**Review by simonyangevorg — DISMISSED** [review]

**Review by LusineHovs — APPROVED** [review]

File: src/ServiceManager/Travely.ServiceManager.UnitTests/ActivityManagerTests.cs [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+                    activity.Type.ActivityName = typeName;
+                    activity.Type.Id = activityTypeId;
+
+                    return _mapper.Map<ActivityDbModel>(activity);
+                });
+
+            var activity = new Activity(Mocks.Activity);
+
+            // Act, Assert
+            await Assert.ThrowsAnyAsync<InvalidOperationException>(() => _target.CreateActivityAsync(Mocks.Activity));
```
Comment (tadart) [inline src/ServiceManager/Travely.ServiceManager.UnitTests/ActivityManagerTests.cs:126]: You can separate Act & Assert
```suggestion
            Task act() => _target.CreateActivityAsync(Mocks.Activity);

            // Assert
            await Assert.ThrowsAnyAsync<InvalidOperationException>(act);
```

Comment (tadart) [reply src/ServiceManager/Travely.ServiceManager.UnitTests/ActivityManagerTests.cs:126]: The same for the other parts.

**Review by tadart — APPROVED** [review]

File: src/ServiceManager/Travely.ServiceManager.UnitTests/ActivityManagerTests.cs (hunk 1 above)
Comment (martikyan) [reply src/ServiceManager/Travely.ServiceManager.UnitTests/ActivityManagerTests.cs]: Done

**Review by simonyangevorg — APPROVED** [review]

**Review by tadart — DISMISSED** [review]

**Review by simonyangevorg — APPROVED** [review]

**Review by LusineHovs — DISMISSED** [review]

**Review by simonyangevorg — APPROVED** [review]

**Review by tadart — APPROVED** [review]

**Pull request merged**